{"update":{"11.04":[{"package":"python-ubuntuone-storageprotocol","version":"1.6.1-0ubuntu1.2"}],"12.04":[{"version":"3.0.0-0ubuntu1.1","package":"python-ubuntuone-storageprotocol"}],"11.10":[{"package":"python-ubuntuone-storageprotocol","version":"2.0.1-0ubuntu1.1"}],"10.04":[{"version":"1.2.0-0ubuntu1.1","package":"python-ubuntuone-storageprotocol"}]},"special":"The problem can be corrected by updating your system to the following package versions:\nIn general, a standard system update will make all the necessary changes.","date":"June 06, 2012","info":"ubuntuone-storage-protocol update","threat":["If a remote attacker were able  to perform a man-in-the-middle attack, this flaw could be exploited to  alter or compromise confidential information.."],"CVEs":["CVE-2011-4409"],"usn":"USN-1465-2"}